test/common/tracing/zipkin/tracer_test.cc

```diff
@@ -26,31 +27,36 @@ class TestReporterImpl : public Reporter {
 TEST(ZipkinTracerTest, spanCreation) {
   Network::Address::InstanceConstSharedPtr addr =
       Network::Address::parseInternetAddressAndPort("127.0.0.1:9000");
-  Tracer tracer("my_service_name", addr);
-  MockMonotonicTimeSource mock_start_time;
-  MonotonicTime start_time = mock_start_time.currentTime();
+  Runtime::RandomGeneratorImpl random_generator;
```

Comment:
I can certainly do that; no problem. The reason I used the real RandomGenerator in some cases was that `MockRandomGenerator::random()` always returns 0, therefore it renders some of my checks useless. It would be nice if we could control the value to be returned by `MockRandomGenerator::random()` to some extent, e.g., zero or non-zero.